QUEST_JOBSTEP.tsv

```diff
@@ -6,21 +6,21 @@ QUEST_JOBSTEP_20150317_000005	If you want to receive my teachings then you have
 QUEST_JOBSTEP_20150317_000007	Good. {nl}I hope your new growth leads to you saving the world. 
 QUEST_JOBSTEP_20150317_000008	Highlander Master
-QUEST_JOBSTEP_20150317_000009	I'm so thankful that you want to be the successor of my swordsmanship, but I am suspicious that you wouldn't follow my teachings well.{nl}I will test you by defeating the Bubbes that are occupying the 1st floor of Crystal Mine.
+QUEST_JOBSTEP_20150317_000009	I'm very thankful that you want to be the successor of my swordsmanship, but I am suspicious that you wouldn't follow my teachings well.{nl}I will test you by asking you to defeat the Vubbes that are occupying the 1st floor of Crystal Mine.
 QUEST_JOBSTEP_20150317_000010	Hurry. {nl} It's a simple challenge, but Vubbes are a threat to the Miner's Village.
-QUEST_JOBSTEP_20150317_000011	Good work, you've completed your initiation in becoming a Highlander. {nl} Keep up the hard work and dedication.
+QUEST_JOBSTEP_20150317_000011	Good work, you've completed your initiation to become a Highlander. {nl} Keep up the hard work and dedication.
 QUEST_JOBSTEP_20150317_000012	Pyromancer Master
 QUEST_JOBSTEP_20150317_000013	I can see that you want to learn magic from me, but I can't teach just anyone. {nl}I will see how well you can assist my research and then decide. 
 QUEST_JOBSTEP_20150317_000014	Good work. Thanks to you I can go on to the next step for my research. {nl}I will teach you Pyromancer magic as promised. 
-QUEST_JOBSTEP_20150317_000015	Good, you're here. I need you to defeat the evil spirits I summon. {nl}I will record the unique wavelength that is transmitted in the process. 
-QUEST_JOBSTEP_20150317_000016	I am doing research on the 4th state of matter. {nl}It is impossible without the cooperation of the Bokor Master who deals with the spirits. 
+QUEST_JOBSTEP_20150317_000015	Good, you're here. I need you to defeat the evil spirits I summon. {nl}I will record the unique wavelength that they transmit in the process. 
+QUEST_JOBSTEP_20150317_000016	I am doing research on the 4th state of matter. {nl}It is impossible without the cooperation of the Bokor Master who deals with spirits. 
 QUEST_JOBSTEP_20150317_000017	Yes, that's great. {nl}Tell the Pyromancer Master that we have all the materials we need. 
-QUEST_JOBSTEP_20150317_000018	Good. {nl}That study is being done with the Bokor Master. So go pay her a visit and she'll tell you what to do. 
+QUEST_JOBSTEP_20150317_000018	Good. {nl}This study is being done with the Bokor Master. So go pay her a visit and she'll tell you what to do. 
 QUEST_JOBSTEP_20150317_000019	Cryomancer Master
 QUEST_JOBSTEP_20150317_000020	You want to learn Cryomancer magic from me? {nl}Sure I'll teach you. If you take care of something first! 
-QUEST_JOBSTEP_20150317_000021	The Pyromancer Master is not around. {nl}Well, it's an easy task.
+QUEST_JOBSTEP_20150317_000021	The Pyromancer Master is not around, so it should be an easy task.
```

Comment:
The quest deals with stealing something from the Pyromancer for the Cryomancer job change quest.  It would seem that it is easier because the Pyromancer is not around to stop you.